### ISSUE TITLE
Empty media tag attributes makes media non-playable.

### ISSUE THREAD

**Issue by kabalin** [lucisgit/moodle-media_jwplayer#14]
Currently media tag specific attributes with empty values results in imposibility to play video without obvious error output.
```
<video width="720" height="480" controls="" loop="" title=""><source src="http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4">http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4</video>
```

**Comment by kabalin** [comment]
As the [spec permits using these tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) without value provided, they simply need to be set to "true" so when they are propagated to setup settings, JW7 player is not getting confused.

**Comment by kabalin** [comment]
Fixed in 7-0.5 release.
